dashboard/src/components/AppList/AppList.tsx

```diff
@@ -131,7 +131,7 @@ function AppList() {
         className="margin-t-xl"
       >
         {error ? (
-          <Alert theme="danger">Unable to list apps: {error.message}</Alert>
+          <ErrorAlert>{error}</ErrorAlert>
```

Comment:
Only usage of the new `ErrorAlert` component by now.

Reply:
Great, just to confirm: you mean there are other errors out there (like `there is a problem with this package...`) which are not being handled by this novel `ErrorAlert`, don't you? 
So this is just a kind of proof of concept, no?

Reply:
Exactly, I first wanted the community to review it. If accepted, we can start using it from now on and eventually replace existing `<Alert>` with it.